--- backtrace-0.3.59/Cargo.toml	1970-01-01T00:00:00+00:00
+++ backtrace-0.3.59/Cargo.toml	2021-05-08T18:09:11.692936+00:00
@@ -47,11 +47,6 @@
 edition = "2018"
 
 [[test]]
-name = "accuracy"
-required-features = ["std"]
-edition = "2018"
-
-[[test]]
 name = "concurrent-panics"
 harness = false
 required-features = ["std"]
@@ -97,19 +92,9 @@
 version = "1.0.67"
 
 [features]
-coresymbolication = []
-dbghelp = []
 default = ["std"]
-dladdr = []
 gimli-symbolize = []
-kernel32 = []
 libbacktrace = []
-libunwind = []
 serialize-rustc = ["rustc-serialize"]
 serialize-serde = ["serde"]
 std = []
-unix-backtrace = []
-verify-winapi = ["winapi/dbghelp", "winapi/handleapi", "winapi/libloaderapi", "winapi/memoryapi", "winapi/minwindef", "winapi/processthreadsapi", "winapi/synchapi", "winapi/tlhelp32", "winapi/winbase", "winapi/winnt"]
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.3"
-optional = true
